src/components/Board.vue

```diff
@@ -1,89 +1,129 @@
 <template>
-  <div id="game">
-    <div id="score">0</div>
+  <div id="game" ref="game">
+    <div id="score">
+      {{ this.$store.state.scoreStore.score }}
+    </div>
     <button class="button-start" @click="startGame">start</button>
-    <div id="touch-area">
-      <div id="good">
-        <div id="excellent">
-        </div>
+    <button class="button-start" style="margin-top: 80px" @click="stopGame">stop</button>
+    <div id="good" ref="good">
+      <div id="excellent" ref="excellent">
       </div>
     </div>
+    <arrow
+      v-for="n in storeItems"
+      :key="n.id"
+    >
```

Comment:
```suggestion
    />
```

no need for `</arrow>`